client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -140,19 +155,37 @@ class SiteImporterInputPane extends React.Component {
 					loading: false,
 					importSiteURL: siteURL,
 				} );
+
+				this.props.recordTracksEvent( 'calypso_site_importer_validate_site_done', {
+					site_url: this.state.importSiteURL,
+					supported_content: resp.supported_content,
```

Comment:
Could this be quite large? If so would that be an issue. I have no idea of the implications there but it seems like something we would want to be careful about.

Reply:
For now I don't expect it to be too large. I'm expecting 6-10 entries tops for now.